src/Stryker.Core/Stryker.Core/Reporters/DashboardReporter.cs

```diff
@@ -16,14 +17,14 @@ public partial class DashboardReporter : IReporter
         private readonly StrykerOptions _options;
         private readonly IDashboardClient _dashboardClient;
         private readonly ILogger<DashboardReporter> _logger;
-        private readonly IChalk _chalk;
+        private readonly TextWriter _output;
 
-        public DashboardReporter(StrykerOptions options, IDashboardClient dashboardClient = null, ILogger<DashboardReporter> logger = null, IChalk chalk = null)
+        public DashboardReporter(StrykerOptions options, IDashboardClient dashboardClient = null, ILogger<DashboardReporter> logger = null, TextWriter output = null)
         {
             _options = options;
             _dashboardClient = dashboardClient ?? new DashboardClient(options);
             _logger = logger ?? ApplicationLogging.LoggerFactory.CreateLogger<DashboardReporter>();
-            _chalk = chalk ?? new Chalk();
+            _output = output ?? Console.Out;
             _logger = ApplicationLogging.LoggerFactory.CreateLogger<DashboardReporter>();
```

Comment:
I will remove the double initialization of _logger here, If you would pass it it along now, it will always get overwritten by the Factory instance.